client/imports/ui/pages/account/account.html

```diff
@@ -2,13 +2,53 @@
   <md-card>
     <md-card-content>
       <div class="account-profile-picture">
-        <img src="{{ user.profile.picture }}">
+        <img src="{{ imgsrc }}">
```

Comment:
I would prefer you keep the code that is "supposed" to be working right now rather than insert temporary variables.